Automattic/wp-calypso
# Extensions: WPJM - Move styles to component level for ease of maintenance

This PR separates the `setup` component styles into multiple `style.scss` files in order to improve organizational structure and ease of maintenance.

## Testing

1. Navigate to `/extensions/wp-job-manager/setup/<site-slug>` in your browser.
2. Visit each of the steps in the wizard and ensure that the styling is not broken.
